### PR TITLE
Add misspell in golangci

### PR DESCRIPTION
**Pull request by wuyingjun-lucky**
Signed-off-by: wuyingjun <wuyingjun_yewu@cmss.chinamobile.com>

**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:
For prevent words misspell and imports do not format into three parts correctly
**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: pkg/karmadactl/cmdinit/kubernetes/deployments.go [hunk 1]
```diff
@@ -108,7 +108,7 @@ func (i *CommandInitOption) makeKarmadaAPIServerDeployment() *appsv1.Deployment
 	}
 
 	// Probes
-	livenesProbe := &corev1.Probe{
+	livenessProbe := &corev1.Probe{
```
Comment (XiShanYongYe-Chang) [inline pkg/karmadactl/cmdinit/kubernetes/deployments.go:111]: Was this modification discovered by `misspell`? In fact, I have some doubts that there should be more than that spelling errors in our project, but why didn't they find them?

**Review by XiShanYongYe-Chang — COMMENTED** [review]
Hi @wuyingjun-lucky, thanks for your contribution!

I think you can put `gci` and `misspell` into two pr, let's deal with them one by one. For `gci`, it looks good to me.

File: pkg/karmadactl/cmdinit/kubernetes/deployments.go (hunk 1 above)
Comment (wuyingjun-lucky) [reply pkg/karmadactl/cmdinit/kubernetes/deployments.go:111]: > Was this modification discovered by `misspell`? In fact, I have some doubts that there should be more than that spelling errors in our project, but why didn't they find them?

NO I found livenesProbe at first and try to use misspell to discover it and the result is `memeber` error

> Hi @wuyingjun-lucky, thanks for your contribution!
> 
> I think you can put `gci` and `misspell` into two pr, let's deal with them one by one. For `gci`, it looks good to me.

OK

File: pkg/search/controller.go [hunk 2]
```diff
@@ -157,7 +157,7 @@ func (c *Controller) handleErr(err error, key interface{}) {
 		return
 	}
 
-	klog.Errorf("Error cache memeber cluster %v, %v", key, err)
+	klog.Errorf("Error cache member cluster %v, %v", key, err)
```
[the remark this answers is not in the document]
Comment (wuyingjun-lucky) [reply pkg/search/controller.go:160]: Yes not include `livenesProbe `

**Pull request merged**